Packs/DatadogCloudSIEM/Integrations/DatadogCloudSIEM/DatadogCloudSIEM.py

```diff
@@ -0,0 +1,791 @@
+            - page_size (int): The number of results per page.
+            - limit (int): The maximum number of results to return.
+            - source (str): The source of the tags to retrieve.
+    Returns:
+        CommandResults: The object containing the command results, including the readable output, outputs prefix,
+         outputs key field, and outputs data.
+    """
+    host_name = args.get("host_name")
+    source = args.get("source")
+    page = arg_to_number(args.get("page"), arg_name="page")
+    page_size = arg_to_number(args.get("page_size"), arg_name="page_size")
+    limit = arg_to_number(args.get("limit"), arg_name="limit")
+    limit, offset = pagination(limit, page, page_size)
+    with ApiClient(configuration) as api_client:
+        tags_api = TagsApi(api_client)
+        response = (
+            tags_api.get_host_tags(host_name=host_name)
+            if not source
+            else tags_api.get_host_tags(host_name=host_name, source=source)
+        )
+        tags = response.get("tags", [])
+    if tags:
+        host_tags_list = get_paginated_results(
+            [{"Tags": tag} for tag in tags], offset, limit
+        )
+        host_tags = [obj.get("Tags") for obj in host_tags_list]
+        context_output = {"Tag": host_tags, "HostTag": host_name}
+        readable_output = lookup_to_markdown(
+            host_tags_list, table_header("Host Tags", page, page_size)
+        )
+    else:
+        readable_output = NO_RESULTS_FROM_API_MSG
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=INTEGRATION_CONTEXT_NAME,
+        outputs_key_field="Tag",
+        outputs=context_output if tags else [],
+    )
+
+
+def add_tags_to_host_command(
+    configuration: Configuration, args: Dict[str, Any]
+) -> Union[CommandResults, DemistoException]:
+    """
+     This function adds tags to a specified host in Datadog.
+
+     Args:
+     configuration (Configuration): The configuration object for Datadog.
+     args (Dict[str, Any]): A dictionary containing the following keys:
+     - host_name (str): The name of the host to add tags to.
+     - tags (str or List[str]): The tags to add to the host, separated by commas or provided as a list.
+
+    Returns:
+     CommandResults: The object containing the command results, including the readable output, outputs prefix,
+      outputs key field, and outputs data.
+    """
+    host_name = args.get("host_name")
+    tags = argToList(args.get("tags"), ",")
+
+    body = HostTags(host=host_name, tags=tags)
+
+    with ApiClient(configuration) as api_client:
+        tags_api = TagsApi(api_client)
+        response = tags_api.create_host_tags(host_name=host_name, body=body)
+    if response and response.get("host"):
+        readable_output = "Tags added to host successfully!"
+        output_context = {"Tag": response.get("tags"), "HostTag": response.get("host")}
+    else:
+        readable_output = ERROR_MSG
```

Comment:
Can you give a more elaborate error than something went wrong ? can we get what went wrong from the API ?